code/__HELPERS/mobs.dm

```diff
@@ -94,7 +94,9 @@
 			break
 
 /proc/random_skin_tone()
-	return pick(skin_tones)
+	var/list/valid_skin_tones = skin_tones
+	valid_skin_tones -= "orange"
```

Comment:
I think this is a very snowflake way of doing this. Does the orange skintone need to be specified in the skin_tones list?